package.json

```diff
@@ -32,6 +32,27 @@
     "ext": "js",
     "exec": "babel-node ./src/server/"
   },
+  "eslintConfig": {
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended"
+    ],
+    "rules": {    
+      "react/react-in-jsx-scope": "off",
+      "semi" : ["warn", "always", { "omitLastInOneLineBlock" : true }]
```

Comment:
Turns on warnings (not errors) about missing semicolons. Also allows us to skip semicolons on single-line blocks.

Reply:
Not sure about this one (`react-app` also does not enable this).
Also, I'm not sure our prettier settings allow one-line `{ ... }` blocks like this? I think they will get formatted onto new lines anyway.

Reply:
I'll remove the omitLastInOneLineBlock thing.

We'll need to have a discussion about semicolon usage. I personally prefer to have them but I know that some people don't, which is why I made it a warning and not an error.

Reply:
👌 I also prefer to always have semicolons, otherwise there's a risk of hard-to-trace errors.

I vote we enforce them with ESLint. Especially if our prettier settings will add them anyway (and that's also the default in `react-app` ESLint settings).